src/strings/PowerFxResources.en-US.resx

```diff
@@ -4760,4 +4760,10 @@
     <value>Unsupported untyped/JSON conversion type '{0}' in argument.</value>
     <comment>Error Message shown to user when a unsupported type is passed in type argument of AsType, IsType and ParseJSON functions.</comment>
   </data>
+  <data name="ErrReachedMaxJsonDepth" xml:space="preserve">
+    <value>Maximum depth reached while traversing JSON payload.</value>    
```

Comment:
>Maximum depth reached while traversing JSON payload. [](http://example.com/codeflow?start=11&length=52)

Please add a comment with more details about this, to help translators avoid bugs that have happened in the past (it also helps us understand where it is used). Something like "Error message returned by the JSON function when a document that is too deeply nested is passed to it. The term JSON refers to the data format described in www.json.org".

Same comment for the next string.